internal/pkg/manifest/manifest.go

```diff
@@ -133,16 +133,18 @@ func (m *Manifest) SetColumns(columns []string) {
 	m.modified = true
 }
 
-func (m *Manifest) SetSchemaColumns(columns []string) {
+func (m *Manifest) SetSchemaColumns(columns []string, optionalDataTypes bool) {
```

Comment:
`optionalDataTypes` - I don't know if it's the best naming, it wasn't clear to me whether it turns the types on or off.

Maybe `addTypes bool` ? or `withTypes bool`